pkg/client/monitoring/v1alpha1/types.go

```diff
@@ -125,6 +127,16 @@ type PrometheusStatus struct {
 	UnavailableReplicas int32 `json:"unavailableReplicas"`
 }
 
+// LogSpec defines the configured log messages from the Prometheus logger.
+type LogSpec struct {
```

Comment:
That makes sense, I just know that there is some fuzz going on around logging and as 2.0 is not final yet there may still be changes to it.